Update domain and theme customization docs for SCIM and SSO (#77)
scalekit-inc/docs: docs/scim/domain-and-theme-customization.mdx

---
toc_max_heading_level: 2
title: 'Customizing the Admin Portal in Scalekit'

> description: 'Learn how to customize the Scalekit Admin Portal to match your brand identity. This guide covers setting up a custom domain, and updating the logo, colors, and favicon for a cohesive user experience.'

Reviewer: ```suggestion
description: 'earn how to customize the Scalekit Admin Portal to match your brand identity. This guide covers setting up a custom domain, updating the logo, colors, and favicon to create a cohesive user experience.'
```

keywords:
  [
    'Scalekit',
    'Admin Portal Customization',
    'Custom Domain',
    'CNAME Configuration',
    'Custom Theme',
    'Brand Identity',
    'Logo',
    'Accent Color',
    'Favicon',
  ]
---

# Customize the Admin Portal

<Subtitle>

>   Configure your Admin Portal UI to match your brand and design needs.

Reviewer: ```suggestion
Customize your Admin Portal UI to align with your brand identity and design preferences```

</Subtitle>

> Customizing your Admin Portal ensures that the interface aligns with your application's brand identity. Follow this guide to update your custom domain, upload your logo, adjust colors, and set a favicon—all of which create a seamless user experience.

Reviewer: ```suggestion
Customizing your admin portal ensures that its interface matches your application's brand identity.

Follow this guide to configure your custom domain, upload your logo, adjust colors, and add a favicon, creating a seamless user experience
```

## Custom Domain

> Custom domains enable you to offer a fully branded experience. By default, Scalekit assigns a unique endpoint URL, but you can replace it via CNAME configuration. This new URL becomes the base path for:

Reviewer: ```suggestion
Custom domains allow you to provide a fully branded experience. By default, Scalekit assigns a unique endpoint URL, but you can replace it using CNAME configuration. This new URL serves as the base path for:```

> - The Admin Portal where your customers recognize your brand.
> - SSO and SCIM configuration details for setting up connections to Identity Providers.
> - The Scalekit REST API Endpoints used by your applications.

Reviewer: ```suggestion
- The admin portal, where customers interact with your brand.
- SSO and SCIM configuration details for setting up connections to Identity Providers.
- The Scalekit REST API endpoints used by your applications.
```

- Follow the Custom Domains steps in the **Customization > Custom Domain** section in your Dashboard.
- Add a CNAME record in your domain’s DNS settings. For example, if your project’s default domain is <SimpleCode>https:<span></span>//yourapp.scalekit.com</SimpleCode>, then:
  - Create a CNAME record for <SimpleCode>auth.yourapp.com</SimpleCode> that resolves to <SimpleCode>yourapp.scalekit.com</SimpleCode>.
  - Use the recommended TTL value to ensure changes propagate quickly.
  - Verify that no extra domain names are appended by your DNS registrar. If extra parts are added (e.g., <SimpleCode>auth.yourapp.com.yourapp.com</SimpleCode>), remove the redundant portion by using only the appropriate prefix (e.g., <SimpleCode>auth</SimpleCode>).

## Custom Theme

Personalize your Admin Portal further by matching it to your brand. The following elements can be customized:

- **Logo:** Upload and display your company logo.
- **Colors:** Adjust the accent color to harmonize with your brand palette.
- **Favicon:** Set a favicon that represents your brand identity.

These customization options help ensure that the Admin Portal feels like an integrated extension of your application.

:::note

> If you need to disable the option to change the enterprise connection or disable an already enabled connection, please contact support at support@scalekit.com.

Reviewer 2: What does this mean? @saif-at-scalekit I didn't understand the context.

Author: This is a experiment.

**Problem Statement**

- The Connection Settings are currently feature gated. How do future users get informed about this or consider if they need this? 
- It also acts like a acknowledgement that this is possible.

What do you think? @ravibits
